### DISCUSSION TITLE
Best approach to updating CEF

### DISCUSSION THREAD

**Discussion by thnewlands** [Voltstro-Studios/UnityWebBrowser#89] [General]
Considering the long term development of tools with this repo -- what's the best way to keep CEF up to date to deal with potential security holes?

**Comment by Voltstro** [comment]
I just periodically update our fork of [CefGlue](https://gitlab.com/Voltstro-Studios/WebBrowser/cefglue) using the gen tools. Most of the time all I have to do is update the `include` files and run the script, but sometimes the manually written C# side has to have some parts added/removed/changed. 

(Also, the main difference with our fork is that the repo has lot less stuff in it, and is a more newer CEF version.)

Then dependabot (or me) will update the submodule. I myself will only usually manually update the submodule if I know there is a breaking change in CefGlue.

**Reply by thnewlands** [reply]
Thanks!! Is dependabot building the binaries used in the packages too? Or is that something you do manually?

**Reply by Voltstro** [reply]
Which binaries? CEF? Because the CEF binaries are just downloaded from the automatic CEF builds site by using one of the scripts.

**Reply by thnewlands** [reply]
Whoa!! Nice! Thank you that answers my question!!
